src/components/modules/FullScreen/FullScreen.view.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+    <slot />
+  </div>
+</template>
+
+<script>
+const OVERLAY_CLASSNAME = 'full-screen-overlay';
+
+export default {
+  name: 'FullScreenView',
+
+  props: {
+    isFullScreen: {
+      type: Boolean,
+      required: true,
```

Comment:
but it should be required, can't see any reason for default value for `Fullscreen` component,
because this is main logic of this component and if you does not pass `isFullScreen` prop, 
maybe you are not correct using this component